compiler/damlc/tests/daml-test-files/InterfaceNoExerciseRequiring.daml

```diff
@@ -25,7 +25,7 @@ interface Token where
 interface SubToken requires Token where
   viewtype EmptyInterfaceView
 
--- @ERROR range=31:3-32:26; Possible Daml-specific reason for the following type error: The choice ‘GetRich’ belongs to interface ‘Token’ which ‘SubToken’ implements.
+-- @ERROR range=31:3-32:26; Possible Daml-specific reason for the following type error: Tried to exercise a choice ‘GetRich’ on interface ‘SubToken’ The choice ‘GetRich’ belongs to interface ‘Token’ which ‘SubToken’ implements.
```

Comment:
I'd avoid the word "implements"; rather, `SubToken` _requires_ `Token`. 
This should also instruct the user to cast the contract id to the right type with `toInterfaceContractId`.

Reply:
Yeah, I noticed while writing this up that the messages could be better formatted as:
- what went wrong
- why
- how to fix it

To avoid growing the GHC-side PR from already a big 150, I'll start another issue & PR

Reply:
I like how that sounds, thank you Dylan